Unicode/makeutype.c

```diff
@@ -177,15 +187,53 @@ static void FigureAlternates(long index, char *apt, int normative) {
 	alts[alts[index][0]][0] = index;
 }
 
-static void processAssignment(long index,char *pt) {
+static int processAssignment(long index,char *pt,long *flg) {
     static long first=-1;
     long i;
 
-    if ( index>0x11ffff )
-return;
+    if ( index<0 ) return( -1 );
+    if ( index>0x11ffff ) return( 0 );
     ++pt;							/* move past semicolon */
     if ( *pt!='<' ) {
 	assignedcodepoints[index/32] |= (1<<(index%32));	/* This Unicode char is visible */
+	/* collect ligatures, vulgar fractions and other fractions */
+	if ( strstr(pt,"LIGATURE") ) {
+	    /* This codepoint index is a ligature */
+	    /* fprintf( stderr, "ligature[%d]=U+%X, = %s\n", lgm, index, pt ); */
+	    if ( lgm >= LG_MAX ) {
+		fprintf( stderr, LgFrcTooMany, "ligatures", "ligature", lgm, index );
+		return( 5 );
+	    }
+	    ligature[lgm] = index;
+	    if ( index < MAXC ) {
+		*flg |= _LIG_OR_FRAC; lgc++;
```

Comment:
...was thinking about MAXC at the time, but linguistically probably makes more sense to call it lg16 as it's better related to uint16 I suppose. If no reply, will change to lg16, ...and comment to point-out as within uint16 range since MAXC was smaller in the past.